imsv-docs-docusaurus/openapi/endpoints/login/auth-token-exchange.yaml

```diff
@@ -0,0 +1,49 @@
+post:
+  tags:
+    - authentication
+  summary: Auth Token Exchange
+  description: |
+    The token endpoint returns a new access token for a user without requiring the user to re-authenticate.
```

Comment:
we should mention somewhere that theres a 60 grace period on the token when its invalidated